mLRS/CommonTx/crsf_interface_tx.h

```diff
@@ -172,8 +172,8 @@ bool tTxCrsf::transmit_start(void)
 
 // a frame is sent every 4 ms, frame length is max 64 bytes
 // a byte is 25 us
-// gaps between frames are 1 ms or so
-#define CRSF_PARSE_NEXTCHAR_TMO_US  500
+// gaps between frames are 3 ms or so
+#define CRSF_PARSE_NEXTCHAR_TMO_US  2200 // Lots of jitter when polled.
```

Comment:
this makes me nervous
what is this needed for??

Reply:
Oops.  It made me nervous too which is why I switched both full and half duplex to use the receive isr.  This can go back to 500.